pkg/controller/mutators/stats_reporter_test.go

```diff
@@ -0,0 +1,83 @@
+package mutators
+
+import (
+	"testing"
+	"time"
+
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+func TestReportMutatorIngestionRequest(t *testing.T) {
+	expectedTags := map[string]string{
+		"status": "active",
+	}
+
+	const (
+		expectedDurationValueMin         = time.Duration(1 * time.Second)
+		expectedDurationValueMax         = time.Duration(5 * time.Second)
+		expectedDurationMin      float64 = 1
```

Comment:
nit: `expectedDurationMin = 1.0` for the same effect and less-wonky spacing.

Reply:
Fixing